v4-client-py/v4_client_py/clients/composer.py

```diff
@@ -155,8 +155,7 @@ def compose_msg_cancel_order(
         )
         return MsgCancelOrder(
             order_id=order_id, 
-            good_til_block=good_til_block,
-            good_til_block_time=good_til_block_time
+            good_til_block=good_til_block
```

Comment:
To avoid overwriting the `good_til_block` attribute, the `good_til_block_time` attribute is unset.

This temporarily breaks stateful order cancellations, the tradeoff is considered worth it since the Python client shouldn't be used for stateful order placements or cancellations.

Reply:
Is there a discussion thread somewhere? Do we have a tracking item fix stateful order cancellations?

Reply:
It's in the slack thread. This python client isn't used for stateful order cancellations at all, it should only be used by programmatic traders.